app/src/main/java/pl/droidevs/books/appHelper/colors.java

```diff
@@ -0,0 +1,66 @@
+package pl.droidevs.books.appHelper;
+
+import android.graphics.Bitmap;
+import android.support.v4.graphics.ColorUtils;
+import android.support.v7.graphics.Palette;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by micha on 18.12.2017.
+ */
+
+public class colors {
```

Comment:
To na co @asvid zwracał uwagę to nazwy paczek z małych czyli zamiast appHelper -> apphelper
I definitywnie nazwy klas w Javie z dużej! Czyli zamiast colors -> Colors

Reply:
Do tego w nazwach pakietów nie używamy camel case: http://www.oracle.com/technetwork/java/codeconventions-135099.html